test/ngsetup_test.go

```diff
@@ -403,7 +404,8 @@ func amfConfig(testID string) error {
 			SupportDnnList: []string{
 				"internet",
 			},
-			NrfUri: "http://127.0.0.10:8000",
+			NrfUri:     "http://127.0.0.10:8000",
+			NrfCertPem: "cert/nrf.pem",
```

Comment:
remove `test/cert` and change the cert path in config:
`NrfCertPem: "../cert/nrf.pem"`